test/functional/combine_logs.py

```diff
@@ -81,10 +80,11 @@ def read_logs(tmp_dir):
     for each of the input log files."""
 
     # Find out what the folder is called that holds the debug.log file
-    chain = glob.glob("{}/node0/*/debug.log".format(tmp_dir))
-    if chain:
-        chain = chain[0]  # pick the first one if more than one chain was found (should never happen)
-        chain = re.search(r'node0/(.+?)/debug\.log$', chain).group(1)  # extract the chain name
+    glob = pathlib.Path(tmp_dir).glob('node0/**/debug.log')
+    path = next(glob, None)
+    if path:
+        assert next(glob, None) is None or '/feature_config_args_' in tmp_dir #  more than one debug.log, should never happen
```

Comment:
16973: where is `or '/feature_config_args_' in tmp_dir` part coming from?

Reply:
We have log collecting by CI even if it's succeed, that's an unique feature of dash.
Bitcoin's implementation fails too for functional test "feature_config_args", but it's never an issue for them. So, that's an workaround for that particular functional test because "never should happen" happens in this particular test but Bitcoin doesn't know it. 

(As tested from same revision of Bitcoin collect log scrip lt fails for Bitcoin for this test too just no one noticed it)